.github/workflows/node.js.yml

```diff
@@ -18,6 +18,19 @@ jobs:
         node-version: ['18.x', '20.x']
 
     steps:
+      - name: 🔐 Harden Runner
+        uses: step-security/harden-runner@63c24ba6bd7ba022e95695ff85de572c04a18142 # v2.7.0
+        with:
+          disable-sudo: true
+          egress-policy: block
+          allowed-endpoints: >
+            cdn.contentful.com:443
+            github.com:443
+            registry.npmjs.org:443
+      - name: ⚙️ Checkout code
+        uses: actions/checkout@b4ffde65f46336ab88eb53be808477a3936bae11 # v4.1.1
+        with:
+          persist-credentials: false
       - uses: actions/checkout@v4
       - name: Use Node.js ${{ matrix.node-version }}
         uses: actions/setup-node@v4
```

Comment:
You also need to change the version by a commit here

Reply:
I do not understand this

Reply:
Instead of

`uses: actions/setup-node@v4`

you should use 

`uses: actions/setup-node@60edb5dd545a775178f52524783378180af0d1f8`